src/pages/Login/Login.scss

```
@@ -37,15 +39,13 @@
       display: flex;
       flex-direction: column;
 
-      .loginId,
-      .loginPassword {
+      .loginInputChange {
         padding: 15px 0;
         border-bottom: 2px solid $color-light-gray;
         transition: 0.2s;
       }
 
-      .loginIdInnerText,
-      .loginPasswordInnerText {
+      .loginInput {
         padding: 15px 0;
         border-bottom: 2px solid $color-middle-gray;
         transition: 0.2s;
```

바뀌는 부분만 따로 작성하시면 될 것 같습니다.

```suggestion
      .loginInput {
        padding: 15px 0;
        border-bottom: 2px solid $color-middle-gray;
        transition: 0.2s;
        
        &.lightBorder {
          border-bottom: 2px solid $color-light-gray;
        }
      }
```

---

하나하나 작성하지 않아도 되는군요 감사합니다!